### PR TITLE
dcrsqlite: remove dead code and swap wrongly positioned args

### PR DESCRIPTION
**Pull request by JFixby**

### CODE REVIEW

File: db/dcrsqlite/sqlite.go [hunk 1]
```diff
@@ -703,13 +690,13 @@ func (db *DB) RetrieveBlockSizeRange(ind0, ind1 int64) ([]int32, error) {
 	}
 	if N < 0 {
 		return nil, fmt.Errorf("Cannot retrieve block size range (%d<%d)",
-			ind1, ind0)
+			ind0, ind1)
```
Comment (chappjc) [inline db/dcrsqlite/sqlite.go]: This one was saying why, so the message here was right.

Comment (JFixby) [reply db/dcrsqlite/sqlite.go]: How about to change it to the `"Cannot retrieve block size range [%d,%d]" `?

**Comment by JFixby** [comment]
Found and fixed 3 more cases.
Also switched the `<` sign

File: db/dcrsqlite/sqlite.go (hunk 1 above)
Comment (chappjc) [reply db/dcrsqlite/sqlite.go]: Sure, that's fine.

**Pull request merged**